sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/InMemoryCheckpointStore.cs

```diff
@@ -195,7 +195,7 @@ public InMemoryCheckpointStore(Action<string> logger = null)
                     PartitionId = data.PartitionId,
                     StartingPosition = EventPosition.FromOffset(data.StartingPosition.Offset.Value, false),
                     ClientIdentifier = data.ClientIdentifier,
-                    LastModified = DateTimeOffset.Parse(data.LastModified)
+                    LastModified = (DateTimeOffset.TryParse(data.LastModified, out var lastModified) ? lastModified : default)
```

Comment:
Thank you for this.  We're seeing weird intermittent test failures due to parsing errors here that I haven't yet attempted to run down.

Reply:
No problem, I saw the error once in the pipeline on this PR, but I haven't seen it again otherwise. I'm not actually totally sure why it only happens sometimes.